Gemfile.lock

```diff
@@ -404,18 +426,20 @@ GEM
       rack
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
-    rails (5.2.6)
-      actioncable (= 5.2.6)
-      actionmailer (= 5.2.6)
-      actionpack (= 5.2.6)
-      actionview (= 5.2.6)
-      activejob (= 5.2.6)
-      activemodel (= 5.2.6)
-      activerecord (= 5.2.6)
-      activestorage (= 5.2.6)
-      activesupport (= 5.2.6)
+    rails (6.0.3.6)
+      actioncable (= 6.0.3.6)
+      actionmailbox (= 6.0.3.6)
```

Comment:
The inclusion of this module in Rails core is a point of some contention in the Rails community :sweat_smile: 

It's only useful if you plan on running an online email service (`*cough* hey.com *cough*`), which 99% of Rails apps do not. It's quite large and I think some of it is enabled by default. We should have a quick look if there's any configs we need to add to **disable it** (I think there might be).

Reply:
Yeah ok, we need to update `application.rb` and remove `require 'rails/all'` and replace it with a list of require statements for all the modules _except_ `ActionMailbox`...

Otherwise we'll be loading a lot of pointless code, and it even **eats extra memory** (doing a job we don't need) :see_no_evil:

Reply:
I made a PR here: https://github.com/luisramos0/openfoodnetwork/pull/11

Reply:
yes, nice, thanks :+1: